README.md

```diff
@@ -40,3 +40,49 @@ interceptor := grpc.UnaryInterceptor(
     }),
 )
 ```
+
+### Logtracing
+
+Used with [tags](`github.com/grpc-ecosystem/go-grpc-middleware/tags`) and [logging](`github.com/grpc-ecosystem/go-grpc-middleware/logging`), allows you to log out a trace and span ID for each call.
+
+```go
+// where 'l' is a log entry
+interceptor := grpc.UnaryInterceptor(
+    grpc.UnaryInterceptor(
+        grpcMiddleware.ChainUnaryServer(
+            grpcTags.UnaryServerInterceptor(),
+            grpcLogTracing.UnaryServerInterceptor(),
+            grpcLogging.UnaryServerInterceptor(l),
+        ),
+    ),
+)
+```
+
+### ctx extractor
+
+Also used with [logging](`github.com/grpc-ecosystem/go-grpc-middleware/logging`), allows you to pass in a function that extracts a string from context. Typically used for functional requirements, like a user ID, or whatever else may be kept in context.
+
+```go
+// where 'l' is a log entry
+// and 'enricher' is a function that extracts a string from context
+interceptor := grpc.UnaryInterceptor(
+    grpc.UnaryInterceptor(
+        grpcMiddleware.ChainUnaryServer(
+            grpcExtractor.UnaryServerInterceptor(enricher),
+            grpcLogging.UnaryServerInterceptor(l),
+        ),
+    ),
+)
+```
+
+### Canceled
+
+Used to capture canceled requests. Alternative it to wrap error codes all the way up, which looks a bit worse, this provides a cheap middleware solution.
```

Comment:
I would give an example of this. Is quite difficult to understand what it does exactly. Just from those words. I would also detail that it is aimed at nested calls.